### ISSUE TITLE
Could we do better to distinguish which user inputs are strings vs which are not?

### ISSUE THREAD

**Issue by ruchirK** [MaterializeInc/materialize#2455]
e.g. in CREATE SINK https://materialize.io/docs/sql/create-sink/ the sink_name and item_name are not strings, the other bits of user input are strings

I'm not sure if the rule is as simple as "things that are names for Materialize are not strings, everything else is a string" and even if that is the rule I don't think its stated in the docs?

it was a bit confusing to me earlier with create sinks and I could see it being confusing for other users?

**Comment by benesch** [comment]
> I'm not sure if the rule is as simple as "things that are names for Materialize are not strings, everything else is a string" and even if that is the rule I don't think its stated in the docs?

Yes, that's the rule. The syntax diagrams should definitely be calling that out.
